config/condor_mapfile

```diff
@@ -1,6 +1,6 @@
 GSI "^\/DC\=com\/DC\=DigiCert-Grid\/O=Open Science Grid\/OU\=Services\/CN\=(host\/)?([A-Za-z0-9.\-]*)$" \2@daemon.opensciencegrid.org
 GSI "^\/DC\=DigiCert-Grid\/DC\=com\/O=Open Science Grid\/OU\=Services\/CN\=(host\/)?([A-Za-z0-9.\-]*)$" \2@daemon.opensciencegrid.org
 GSI (.*) GSS_ASSIST_GRIDMAP
-GSI (.*) anonymous@gsi
+GSI (/CN=[-.A-Za-z0-9/= ]+) \1@unmapped.opensciencegrid.org
```

Comment:
Hi Brian,

Actually, I think it's complaining about the "=", isn't it?

This is just supposed to match any reasonable pilot CN.

Brian